pkg/pdutil/pd_api_client_test.go

```diff
@@ -95,3 +100,53 @@ func TestListGcServiceSafePoint(t *testing.T) {
 	require.Nil(t, err)
 	mockClient.testServer.Close()
 }
+
+// LabelRulePatch is the patch to update the label rules.
+// NOTE: This type is exported by HTTP API. Please pay more attention when modifying it.
+// Copied from github.com/tikv/pd/server/schedule/labeler
+type LabelRulePatch struct {
+	SetRules    []*LabelRule `json:"sets"`
+	DeleteRules []string     `json:"deletes"`
+}
+
+// LabelRule is the rule to assign labels to a region.
+// NOTE: This type is exported by HTTP API. Please pay more attention when modifying it.
+// Copied from github.com/tikv/pd/server/schedule/labeler
+type LabelRule struct {
+	ID       string        `json:"id"`
+	Index    int           `json:"index"`
+	Labels   []RegionLabel `json:"labels"`
+	RuleType string        `json:"rule_type"`
+	Data     interface{}   `json:"data"`
+}
+
+// RegionLabel is the label of a region.
+// NOTE: This type is exported by HTTP API. Please pay more attention when modifying it.
+// Copied from github.com/tikv/pd/server/schedule/labeler
+type RegionLabel struct {
```

Comment:
Why not import the `github.com/tikv/pd/server/schedule/labeler` directly? I saw it's a public type.

See: https://github.com/tikv/pd/blob/master/server/schedule/labeler/rules.go#L32

Reply:
It make TiCDC depends on pd server which may bloat TiCDC binary size and dependency graph.

Reply:
Actually, we have depends on it. see: https://github.com/pingcap/tiflow/blob/master/go.mod#L77

Reply:
Tried, failed to compile tests
```shell
make tidy     
go mod tidy
./tools/check/check-tidy.sh
go: finding module for package google.golang.org/grpc/naming
github.com/pingcap/tiflow/pkg/pdutil tested by
        github.com/pingcap/tiflow/pkg/pdutil.test imports
        github.com/tikv/pd/server/schedule/labeler imports
        github.com/tikv/pd/server/storage/endpoint imports
        go.etcd.io/etcd/clientv3 tested by
        go.etcd.io/etcd/clientv3.test imports
        go.etcd.io/etcd/integration imports
        go.etcd.io/etcd/proxy/grpcproxy imports
        google.golang.org/grpc/naming: module google.golang.org/grpc@latest found (v1.48.0), but does not contain package google.golang.org/grpc/naming
make: *** [Makefile:292: tidy] Error 1
```

We have to upgrade etcd version in PD or patch etcd in TiCDC side, I think it's not worth to do so, only complicates already complexed dependency.

Reply:
Sadly